superset/db_engine_specs/postgres.py

```diff
@@ -310,7 +310,7 @@ def get_catalog_names(
 SELECT datname FROM pg_database
 WHERE datistemplate = false;
             """
-            ).fetchall()
```

Comment:
Curious about why this had to change?

Reply:
@Antonio-RiveroMartnez in this case it's redundant, the expression generator will implicitly call `fetchall()` here. Eg:

```python
>>> from sqlalchemy import create_engine
>>> engine = create_engine("shillelagh://")
>>> url = "https://docs.google.com/spreadsheets/d/1LcWZMsdCl92g7nA-D6qGRqg1T5TiHyuKJUY1u9XAnsk/edit#gid=0"
>>> engine.execute(f'SELECT * FROM "{url}"').fetchall()
[('BR', 2), ('BR', 4), ('ZA', 7), ('CR', 11), ('CR', 11), ('FR', 100), ('AR', 42)]
>>> [row for row in engine.execute(f'SELECT * FROM "{url}"')]
[('BR', 2), ('BR', 4), ('ZA', 7), ('CR', 11), ('CR', 11), ('FR', 100), ('AR', 42)]
```